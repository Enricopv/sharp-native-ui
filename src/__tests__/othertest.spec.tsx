import "@testing-library/jest-dom/extend-expect"
import "jest-styled-components"
import * as React from "react"
import * as renderer from "react-test-renderer"
import { OtherButton } from "../"

test("Shows the children of Other Button", () => {
  const tree = renderer.create(<OtherButton>Other Button</OtherButton>).toJSON()
  expect(tree).toMatchSnapshot()
})
