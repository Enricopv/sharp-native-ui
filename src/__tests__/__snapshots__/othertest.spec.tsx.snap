// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shows the children of Other Button 1`] = `
<div
  className="css-text-901oao r-backgroundcolor-y5gxn7 r-color-jwli3a"
  dir="auto"
>
  Other Button
</div>
`;
